.rubocop.yml

```diff
@@ -1,9 +1,11 @@
 require:
+  - rubocop-rspec
```

Comment:
I'm pretty sure that the latest version of `rubocop-rspec` (currently 3.5.0) supports being loaded as a plugin. There is a "Pluginfy RuboCop RSpec" in the release notes [here](https://github.com/rubocop/rubocop-rspec/releases/tag/v3.5.0). Is there a reason that you are leaving `rubocop-rspec` under `require`, rather than loading it with the other `plugins`?

Reply:
yes - it requires updating `rspec-rails` to a version that's incompatible with `rails-6.1`.